reactDashboard/src/components/ChartDropdown/ChartDropdown.css

```diff
@@ -31,13 +31,25 @@
 .selected-chart:hover {
   background-color: #ec6730;
   border-color: #ec6730;
-  color: #ffffff;
+  color: white;
+}
+
+.darkmode .selected-chart:hover {
+  background-color: rgb(64, 64, 64);
+  border-color: rgb(64, 64, 64);
+  color: white;
```

Comment:
This is the same as light mode - can it be deleted?

Reply:
✅

Reply:
I meant just the line commented, not the whole block. Are you ok losing background-color and border-color?

Reply:
this now contains a specific .darkmode color to remove the wave contrast error

Reply:
Not for .darkmode .selected-chart:hover (that I can see)

Reply:
I still need to push these changes

Reply:
removed hover for .selected-chart